### PR TITLE
[stable/prometheus-cloudflare-exporter] Add chart

### PR DESCRIPTION
**Pull request by javad-hajiani**
## Checklist

- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)
- [x] Github actions are passing

### CODE REVIEW

File: stable/prometheus-cloudflare-exporter/values.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+image:
+  repository: osig/prometheus-cloudflare-exporter
+  tag: 1.2.0
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
```
Comment (nyambati) [inline stable/prometheus-cloudflare-exporter/values.yaml:16]: You don't have service account config, rename this to just serviceAccountName

**Review by max-rocket-internet — CHANGES_REQUESTED** [review]
You don't have any annotations included for prom exporters, so prom will never find the `/metrics` endpoint.

I think it's better to include the required pod annotations by default and make the service optional, like here: https://github.com/deliveryhero/helm-charts/blob/master/stable/prometheus-soti-mobicontrol-exporter/values.yaml#L55-L66

**Review by nyambati — CHANGES_REQUESTED** [review]
Include pod and service annotations for prometheus 
https://github.com/deliveryhero/helm-charts/blob/3bce1203e9bc24ff4113e8d8b8b2450a80522179/stable/aws-service-events-exporter/values.yaml#L19

**Comment by max-rocket-internet** [comment]
You also need to include the options from here: https://github.com/wehkamp/docker-prometheus-cloudflare-exporter#try-it

For example, I would put `AUTH_KEY` in a secret, like here: https://github.com/deliveryhero/helm-charts/blob/master/stable/prometheus-soti-mobicontrol-exporter/templates/secret.yaml

**Comment by javad-hajiani** [comment]
> You also need to include the options from here: https://github.com/wehkamp/docker-prometheus-cloudflare-exporter#try-it
> 
> For example, I would put `AUTH_KEY` in a secret, like here: https://github.com/deliveryhero/helm-charts/blob/master/stable/prometheus-soti-mobicontrol-exporter/templates/secret.yaml

I've wanted to just pass them as environment variable to deployment which could be more flexible

File: stable/prometheus-cloudflare-exporter/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "prometheus-cloudflare-exporter.fullname" . }}
+  labels:
+{{ include "prometheus-cloudflare-exporter.labels" . | indent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: 80
```
Comment (max-rocket-internet) [inline stable/prometheus-cloudflare-exporter/templates/service.yaml:10]: I think it's conventional for exporters to use the same port

```suggestion
    - port: 9199
```

**Review by max-rocket-internet — APPROVED** [review]
Nice!

**Comment by javad-hajiani** [comment]
> Nice!

Thanks for giving awesome suggestions

**Review by nyambati — APPROVED** [review]

**Pull request merged**